.github/actions/upload-flaky-tests/action.yml

```diff
@@ -0,0 +1,44 @@
+name: Report flaky test
+description: Finds flaky tests and creates issues
+
+runs:
+  using: composite
+  steps:
+    - id: flaky-tests
+      name: Find flaky tests
+      if: github.repository == 'stianst/keycloak'
```

Comment:
Running this only in the `stianst` org is probably outdated. 

Should this run only on the main repo in all the branches? I'm considering it should exclude PRs, as those would contain code which is not in the main line.